misc/python/materialize/output_consistency/execution/evaluation_strategy.py

```diff
@@ -26,19 +26,31 @@
     TableColumnByNameSelection,
 )
 
+DUMMY_EVALUATION_STRATEGY_ID = "DUMMY"
```

Comment:
This being strings kind of defeats using mypy type hints. You could use an enum instead: https://docs.python.org/3/library/enum.html https://mypy.readthedocs.io/en/stable/literal_types.html#enums

Reply:
✅